app/views/record/_sidebar.html.erb

```diff
@@ -1,12 +1,14 @@
-<h4>Sidebar</h4>
+<div class="bit">
+  <h3 class="title">Sidebar</h4>
 
-<ul>
-  <li>Advanced Search</li>
-  <li>Save to...</li>
+  <ul>
+    <li>Advanced Search</li>
```

Comment:
Do we know what these functions actually do yet? And if so, are there tickets for making them do that?

Reply:
Oh yes, I suppose I could adjust this to be at least closer to real... brb.